src/main/java/com/Elessar/app/client/MyClient.java

```diff
@@ -19,6 +23,52 @@ public MyClient(URL hostURL) {
         this.hostURL = hostURL;
     }
 
+    public void logOn(String userName, String password) {
+        final LogonRequest.Builder logonReq= LogonRequest.newBuilder();
+        logonReq.setName(userName).setPassword(password);
```

Comment:
Even though we are not too concerned about the "proper" way of handling password, let's at least do some sort of encoding here so at least we don't store plain text password :)
Don't forget to decode at the server side before you compare passwords